resources/CMakeLists.txt

```diff
@@ -32,6 +32,24 @@ endif()
 if (WITH_SERVER)
   file(GLOB_RECURSE SERVER_RESOURCE_FILES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} server/*)
   set(RESOURCES_FILES ${RESOURCES_FILES} ${SERVER_RESOURCE_FILES})
+
+  # Landingpage
+  file(GLOB_RECURSE LANDINGPAGE_SOURCE_FILES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} server/src/landingpage/*)
+  list(REMOVE_ITEM $(LANDINGPAGE_SOURCE_FILES) "*landingpage.stamp")
+
+  add_custom_target (yarn-target ALL DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/server/src/landingpage/landingpage.stamp)
+
+  add_custom_command(
+    POST_BUILD
+    OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/server/src/landingpage/landingpage.stamp
+    DEPENDS ${LANDINGPAGE_SOURCE_FILES}
+    COMMAND ${CMAKE_COMMAND} -E touch ${CMAKE_CURRENT_SOURCE_DIR}/server/src/landingpage/landingpage.stamp
+    COMMAND cd ${CMAKE_CURRENT_SOURCE_DIR}/server/src/landingpage && yarn install --frozen-lockfile && yarn build
```

Comment:
Would it be possible to copy first and build after, this way the source tree would be left untouched
Same for the landingpage.stamp, this should go into the build dir and not into the source dir.

Reply:
Good idea! Yes, that's possible, done in https://github.com/qgis/QGIS/pull/46643/commits/34f7d640c5b5b0f4b19472906b503371685b347b.